scripts/config.pl

```diff
@@ -177,7 +171,10 @@
 
```

Comment:
A bug that I should fix while I'm at it: the regexps in `$exclude_re` and `$no_exclude_re` should be anchored. As they stand, if an option name happens to be a prefix of another one, it'll be included.